tests/io/v3/vm/scans/test_scans_api.py

```diff
@@ -0,0 +1,911 @@
+                                'inputs': [
+                                    {
+                                        'type': 'radio-group',
+                                        'id': 'report_paranoia',
+                                        'label': 'Override normal accuracy',
+                                        'default': 'Normal',
+                                        'options': [
+                                            'Normal',
+                                            'Avoid False alarms',
+                                            'Paranoid (more False alarms)'
+                                        ],
+                                        'optionsLabels': [
+                                            '',
+                                            'Avoid potential False alarms',
+                                            'Show potential False alarms'
+                                        ]
+                                    },
+                                    {
+                                        'type': 'checkbox',
+                                        'id': 'thorough_tests',
+                                        'label': 'Perform thorough tests (may disrupt your network or impact scan speed)',
+                                        'default': 'no'
+                                    }
+                                ],
+                                'title': 'Accuracy',
+                                'name': 'accuracy'
+                            }
+                        ]
+                    }
+                ],
+                'sections': []
+            },
+            'advanced': {
+                'inputs': 'None',
+                'modes': {
+                    'id': 'advanced_mode',
+                    'name': 'mode',
+                    'type': 'ui_radio',
+                    'default': 'Custom',
+                    'options': []
+                },
+                'title': 'Advanced',
+                'groups': [],
+                'sections': [
+                    {
+                        'inputs': [],
+                        'title': 'General Settings',
+                        'name': 'advanced'
+                    }
+                ]
+            },
+            'discovery': [],
+            'report': {}
+        },
+        'name': 'basic'
+    }
+    return editor_resp_data
+
+
+@responses.activate
+def test_details(api):
+    scan_id = '731a8e52-3ea6-a291-ec0a-d2ff0619c19d7bd788d6be818b65'
+    editor_resp_data = get_editor_details_data()
+
+    details_resp = {
+        'settings':
+            {'name': 'Example Embedded Cred Scan',
+             'report_paranoia': 'Normal',
+             'thorough_tests': 'no',
+             'enabled': True,
+             'rrules': None,
+             'timezone': None,
+             'starttime': None
+             },
+        'uuid': '731a8e52-3ea6-a291-ec0a-d2ff0619c19d7bd788d6be818b65'
```

Comment:
it should be 'id' here and at other instances as well uuid should be replaced with id.

Reply:
some uuids needs to be in similar manner because currently it uses version 2 endpoints. once we migrate it to v3 we will fix this.